### PR TITLE
fixed that automation test doesn't work

### PR DESCRIPTION
**Pull request by jinwon0325kim**
Issue : (https://github.com/Samsung/ioter/issues/45)

Invalid args in automatinWindow.
When the part that creation of dialog for the automationWindow in the main is modified, args are not modified.

### CODE REVIEW

**Review by sangsoolee — APPROVED** [review]
thanks for fix the bug

File: src/automation/ProcessCmd.py [hunk 1]
```diff
@@ -170,8 +170,8 @@ class ProcessCmd(QThread):
     update_highlight = pyqtSignal(int)
     complete_autotest = pyqtSignal()
     # Class Variable
-    mDevConfFilepath = 'automation/conf/'
-    mScriptFilePath = 'automation/output/'
+    mDevConfFilepath = 'src/automation/conf/'
+    mScriptFilePath = 'src/automation/output/'
```
Comment (junyong-sim) [inline src/automation/ProcessCmd.py:174]: nit.
I think mDevConfFilepath and mScriptFilePath variables are not using, now.
If right, can we remove them ?

**Review by junyong-sim — APPROVED** [review]
LGTM

File: src/automation/ProcessCmd.py (hunk 1 above)
Comment (jinwon0325kim) [reply src/automation/ProcessCmd.py:174]: mDevConfFilepath and mScriptFilePath are used in automation test.

**Pull request merged**